### PR TITLE
chore: Code cleanup

### PR DESCRIPTION
**Pull request by gregor**

### CODE REVIEW

File: app/script/assets/AssetCrypto.js [hunk 1]
```diff
@@ -88,7 +88,7 @@ z.assets.AssetCrypto = (() => {
   function _generate_random_bytes(length) {
     const randomValues = new Uint32Array(length / 4).map(() => libsodium.getRandomValue());
     const ramdonBytes = new Uint8Array(randomValues.buffer);
```
Comment (lipis) [inline app/script/assets/AssetCrypto.js:90]: I can see that we already using `camelCase` and I would recommend using them for new variables.. this change can't happen in one go.. so replacing them here and there shouldn't be a problem :)

File: app/script/calling/SDPMapper.js [hunk 2]
```diff
@@ -84,43 +84,43 @@ z.calling.SDPMapper = {
...
         }
       } else if (sdp_line.startsWith('a=rtpmap')) {
-        if (
-          flow_et.negotiation_mode() === z.calling.enum.SDP_NEGOTIATION_MODE.ICE_RESTART ||
-          (sdp_source === z.calling.enum.SDP_SOURCE.LOCAL && flow_et.is_group)
-        ) {
+        const is_ice_restart = flow_et.negotiation_mode() === z.calling.enum.SDP_NEGOTIATION_MODE.ICE_RESTART;
+        const is_local_sdp = sdp_source === z.calling.enum.SDP_SOURCE.LOCAL;
+
+        if (is_ice_restart || (is_local_sdp && flow_et.is_group)) {
```
Comment (lipis) [inline app/script/calling/SDPMapper.js:123]: for example here.. we could maybe use the `camelCases`.. wdyt?

Comment (gregor) [reply app/script/calling/SDPMapper.js:123]: I have talked to @bennyn and we are going to start with this.

Comment (lipis) [reply app/script/calling/SDPMapper.js:123]: nice!

**Pull request merged**